waiter/docs/parameters.md

```diff
@@ -16,7 +16,7 @@ Additional (optional) parameters that can be set:
 
 |Parameter|Default Value|Valid Values|Description|Guidance|
 |---------|-------------|------------|-----------|--------|
-|`X-Waiter-Allowed-Params`|empty vector|vector of non-empty strings|The environment variables configured as parameters for a tokenized service via `X-Waiter-Param-*` header which will not trigger an on-the-fly mode.|Expose only environment variables which your command can handle safely in this list. It is also a good idea to provide default values for these parameters, e.g. in your `env` configuration.|
+|`X-Waiter-Allowed-Params`|empty|vector of non-empty strings (token) or comma-separated strings (headers)|The environment variables configured as parameters for a tokenized service via `X-Waiter-Param-*` header which will not trigger an on-the-fly mode.|Expose only environment variables which your command can handle safely in this list. It is also a good idea to provide default values for these parameters, e.g. in your `env` configuration.|
```

Comment:
It may not make sense in some cases for a default value.

Reply:
CHanged the message to a suggestion as a location for default values.